main.go

```diff
@@ -22,6 +22,10 @@ func main() {
 	app.Name = path.Base(os.Args[0])
 	app.Author = "Docker Machine Contributors"
 	app.Email = "https://github.com/docker/machine"
+	app.Before = func(c *cli.Context) error {
```

Comment:
Not specifically related to this PR, but if this actually works (this being app.Before) without screwing up, we should move the debug flag to here.